### PR TITLE
fix library_abs set to NOTFOUND; issue #15

### PR DESCRIPTION
**Pull request by amar-laksh**
Here we go, Sorry for the earlier one.

### CODE REVIEW

File: libcurl_vendor/libcurl_vendor-extras.cmake.in [hunk 1]
```diff
@@ -30,6 +30,7 @@ endif()
 # message(STATUS "CURL_INCLUDE_DIRS: ${CURL_INCLUDE_DIRS}")
 
 set(libcurl_vendor_LIBRARIES)
+set(libcurl_abs ${CURL_LIBRARY_DIRS})
```
Comment (davidhuziji) [inline libcurl_vendor/libcurl_vendor-extras.cmake.in]: Hi @amar-laksh, I guess it should be `library_abs` instead of `libcurl_abs`.

**Review by davidhuziji — COMMENTED** [review]
Tried with "library_abs". It works.

**Comment by vilvo** [comment]
`set(library_abs ${CURL_LIBRARY_DIRS})` fixes the issue.

**Comment by sloretz** [comment]
ROS2 CI
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/4219/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/1264/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3492/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/4325/)

**Comment by sloretz** [comment]
Hi @amar-laksh , Thanks for the pull request (and thanks @davidhuziji and @vilvo for testing).

While this does get a cross-compile build of `resource_retriever` to work, it breaks normal builds of `rviz2`. Would you mind investigating? ([jenkins build](https://ci.ros2.org/job/ci_linux/4219/consoleText))

```
[ 83%] Building CXX object CMakeFiles/rviz_default_plugins_object.dir/src/rviz_default_plugins/robot/robot_link.cpp.o
/home/rosbuild/ci_scripts/ws/src/ros2/rviz/rviz_default_plugins/src/rviz_default_plugins/robot/robot_link.cpp:66:42: fatal error: resource_retriever/retriever.h: No such file or directory
compilation terminated.
CMakeFiles/rviz_default_plugins_object.dir/build.make:1046: recipe for target 'CMakeFiles/rviz_default_plugins_object.dir/src/rviz_default_plugins/robot/robot_link.cpp.o' failed
```

Possibly related, during a cross compile build I notice this output during `resource_retriever`. Somewhere along the way `resource_retriever` is trying to link to the content of `${CURL_LIBRARY_DIRS}`.

```
WARNING: Target "resource_retriever" requests linking to directory "/other/tmp/r2_cross/ros2_ws/build/libcurl_vendor/libcurl_install/lib"
.  Targets may link only to libraries.  CMake is dropping the item.
```

**Comment by clalancette** [comment]
This hasn't had a response in a long time, so I'm going to close it out.  Please feel free to reopen and answer @sloretz 's review if you still want to move forward with this.

**Pull request closed without merging**